_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Reporting CLI
+nav_order: 25
+---
+
+# Reporting CLI
+
+You can download OpenSearch Dashboards reports directly through the reporting CLI. You don't need to use the OpenSearch Dashboards or the Reporting plugin to download reports. You choose the file format that you want and specify either PNG, PDF or CSV file format with the `--formats` argument.
+
+You can also send an email with the report file attached to it.
+
+You can connect to OpenSearch with either of the following authentication types:
+
+* Basic – `basic`
+* Cognito – `cognito`
+* SAML – `saml`
+* No auth – `none`
+
+The reporting CLI tool allows you to integrate it into your own workflow or environment so that you can bypass authentication or potential security issues. For example, if you use the reporting CLI tool within an AWS Lambda instance, there are no security issues that would occur if you ran the Reporting plugin in OpenSearch Dashboards. In this case, you would use "No auth" to bypass the authentication process. Specify `--auth none` in your request. To get a list of all options, see [Command-line Options](https://github.com/opensearch-project/dashboards-reporting/reporting-cli/docs/user/Reference Manual.md) in the GitHub Reference Manual.
```

Comment:
Suggestion: "For example, if you use the reporting CLI tool within an AWS Lambda instance, no security issues would occur as long as you run the Reporting plugin in Dashboards."